docs/content/en/v2/api/controls/carousel-view.md

```diff
@@ -0,0 +1,73 @@
+---
+id: "carousel-view"
```

Comment:
Can you prefix all ids with `v2-` please?
Currently a limitation of the website. I would prefer only showing v2 docs, but it's mixing v1 and v2 for now

Reply:
Fixed :)